### PR TITLE
change address to 32bytes

### PR DESCRIPTION
**Pull request by slave5vw**
secp256k1 address style : blake2b-256(pubkey)

### CODE REVIEW

File: crypto/tmhash/hash.go [hunk 1]
```diff
@@ -58,8 +58,9 @@ func NewTruncated() hash.Hash {
 	}
 }
 
-// SumTruncated returns the first 20 bytes of SHA256 of the bz.
+// SumTruncated returns the first 32 bytes of SHA256 of the bz.
+// NOTE: friday changes the length of all addresses to 32 bytes.
 func SumTruncated(bz []byte) []byte {
 	hash := sha256.Sum256(bz)
```
Comment (JoowonYun) [inline crypto/tmhash/hash.go:64]: sha256?

File: crypto/ed25519/ed25519.go [hunk 2]
```diff
@@ -134,7 +134,7 @@ const PubKeyEd25519Size = 32
 // PubKeyEd25519 implements crypto.PubKey for the Ed25519 signature scheme.
 type PubKeyEd25519 [PubKeyEd25519Size]byte
 
-// Address is the SHA256-20 of the raw pubkey bytes.
+// Address is the SHA256-32 of the raw pubkey bytes.
 func (pubKey PubKeyEd25519) Address() crypto.Address {
 	return crypto.Address(tmhash.SumTruncated(pubKey[:]))
```
Comment (JoowonYun) [inline crypto/ed25519/ed25519.go:139]: Isn't it `return crypto.Address(pubkey [:])` as said yesterday?

File: crypto/tmhash/hash.go (hunk 1 above)
Comment (slave5vw) [reply crypto/tmhash/hash.go:64]: it's for ed25519 or bls. not used secp256k1

Comment (psy2848048) [reply crypto/tmhash/hash.go:64]: Confirmed. SumTruncated -> AddressHash -> threashold_pubkey 
SumTruncated is only used in BLS and ED25519

**Pull request merged**